spec/helpers/hearing_helper_spec.rb

```diff
@@ -34,4 +34,26 @@
 
     it { is_expected.to eql('2021-01-19T10:45:15.000Z'.to_datetime) }
   end
+
+  describe '#format_and_dislay_crlf' do
```

Comment:
is this  the old name? it should match the subject anyway
```suggestion
describe '#display_crlf_and_sanitize' do
```

Reply:
Thanks for catching that